demo/ez-browse.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>ez-browse demo</title>
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+    <link rel="import" href="../ez-browse.html">
+
+    <style is="custom-style" include="demo-pages-shared-styles">
+        ez-browse.demoTest:after {
+            content: "StartingLocationId given to the UDW is: '" attr(selected-location-id) "'";
+        }
+    </style>
+  </head>
+    <body>
+        <div class="vertical-section-container centered">
+            <h3>Basic ez-browser demo</h3>
+            <demo-snippet>
+              <template>
+                  <ez-browse selected-location-id="2">
+                      Browse
+                  </ez-browse>
+                  <script>
+                        this.addEventListener('ez:contentDiscover', (e) => {
```

Comment:
`document` would be a bit better than `this`. Also, when receiving the event instead of updating the `ez-browse` it self, you should fill a *log element* added next to the browse element (similar to https://github.com/ezsystems/hybrid-platform-ui-core-components/blob/master/demo/ez-notification.html#L71)